frontend/javascripts/oxalis/model/bucket_data_handling/wkstore_adapter.js

```diff
@@ -145,23 +146,40 @@ export async function requestFromStore(
     createRequestBucketInfo(zoomedAddress, resolutions, fourBit, applyAgglomerates, version),
   );
 
-  return doWithToken(async token => {
-    const { buffer: responseBuffer, headers } = await Request.sendJSONReceiveArraybufferWithHeaders(
-      `${dataUrl}/data?token=${token}`,
-      {
-        data: bucketInfo,
-        timeout: REQUEST_TIMEOUT,
-      },
-    );
-    const missingBuckets = parseAsMaybe(headers["missing-buckets"]).getOrElse([]);
-
-    let resultBuffer = responseBuffer;
-    if (fourBit) {
-      resultBuffer = await decodeFourBit(resultBuffer);
-    }
-
-    return sliceBufferIntoPieces(layerInfo, batch, missingBuckets, new Uint8Array(resultBuffer));
-  });
+  try {
+    return doWithToken(async token => {
+      const stuff = await Request.sendJSONReceiveArraybufferWithHeaders(
+        `${dataUrl}/data?token=${token}`,
+        {
+          data: bucketInfo,
+          timeout: REQUEST_TIMEOUT,
+          showErrorToast: false,
+          doNotInvestigate: true,
```

Comment:
Does `doNotInvestigate: true` make sense? If buckets cannot be served, the chance is high that this is due to a datastore which is down (which would be investigated if doNotInvestigate was false).

Reply:
Thanks for nagging, somehow I thought that the error will be handled earlier when investigating the error. But a look at the code told me that the error is passed on to the upper level (`Promise.reject`)